src/pages/Home.jsx

```diff
@@ -55,61 +29,30 @@ function Home() {
               <ul className="nav nav-pills outline-active">
                 <li className="nav-item">
                   <button
-                    onClick={() => setActiveTag(null)}
+                    onClick={() => setFilters((prevFilters) => ({...prevFilters, tag:null}))}
```

Comment:
Notice here how colocating the state makes you work harder but doesn't give you any benefit.